handbook.md

```diff
@@ -80,6 +80,17 @@ Optionally, include more detailed explanatory text if necessary and wrap it to a
 
 Ensure the description is of the change and not the bug title, e.g. "X now accepts Y when doing Z" rather than "Z throws error".
 
+If the commit that you are adding fixes multiple issues you can follow this example:
+
+```
+Fixes #1234,#5678 - Fix content view yum publish bug
```

Comment:
Please improve this example by not repeating "Fix" and to be more specific (match the style or simply re-use the previous example).